README.rdoc

```diff
@@ -95,7 +95,14 @@ http://reenhanced.com/reflow/git-reflow-start.gif
 
 This sets up a feature branch remotely and brings a local copy to your machine. Yeah, you can do this by hand pretty easily, so skip this command if you want. This is just a handy shortcut with no magic.
 
-  $ git reflow start nh-branch-name
+  git reflow start nh-branch-name
+
+"Git Reflow Start" takes in the name of the new branch name that you want to create your feature on. 
+In addition, it takes in an optional flag of a base branch name. If you don't pass in this parameter, then it defaults to "master". 
+The base branch name is the base branch that you want to base your feature off of. 
+This ensures that everytime you start a new base branch, it will be based off of your latest remote base.
+
+  git reflow start nh-branch-name --base base-branch-name
```

Comment:
I think it is --base or -b followed by a space.

According to iTerm, it works:

```
Dev-Loaner-3s-MacBook-Pro:gitreflow ximengzhu$ git branch
  master
  my_new_branch
  sz-issue/118-no-help-from-git-reflow---help
  sz-issue/141-2xLGTM
  sz-issue/149-Fix-Commit-Message-to-merge-instead-of-close
  sz-issue/162-Update-Rspec
  sz-issue/162-converting-rspecs-to-use-allow
* sz-issue/66-always-reflow-start-from-master
  sz-issue/73+66-configurable-base-branch-refresh-option
  test
  test_2
  test_3
Dev-Loaner-3s-MacBook-Pro:gitreflow ximengzhu$ git reflow start test_4 --base my_new_branch
Switched to branch 'my_new_branch'
From https://github.com/simonzhu24/gitreflow
 * branch            my_new_branch -> FETCH_HEAD
Counting objects: 7, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (7/7), done.
Writing objects: 100% (7/7), 1.10 KiB | 0 bytes/s, done.
Total 7 (delta 5), reused 0 (delta 0)
To https://github.com/simonzhu24/gitreflow.git
 * [new branch]      my_new_branch -> test_4
error: you need to resolve your current index first
(reverse-i-search)`git push': git push -f origin sz-issue/73+66-configurable-base-branch-refresh-option
Dev-Loaner-3s-MacBook-Pro:gitreflow ximengzhu$
Dev-Loaner-3s-MacBook-Pro:gitreflow ximengzhu$ git status
On branch my_new_branch

```